pkg/monitor/api.go

```diff
@@ -54,6 +60,9 @@ func Start(ctx context.Context) (*Monitor, error) {
 	if err := StartKubeAPIMonitoringWithNewConnections(ctx, m, clusterConfig, 5*time.Second); err != nil {
 		return nil, err
 	}
+	if err := StartKubeAPIMonitoringWithNewConnectionsForNodeIPs(ctx, m, clusterConfig, 5*time.Second); err != nil {
```

Comment:
This is the first monitor that needs to be conditional.  If you plan on running as a pod, we don't want to list / watch everything from inside the cluster (we're watching from outside).  You probably need to have a monitor specifically for this, and you should not be running this from outside the cluster.

A proxy connection through the cluster will be disrupted.